### PR TITLE
Select image which will be shown on screen for each cut

### PR DESCRIPTION
**Pull request by namse**
# What I did
- Select image which will be shown on screen for each cut
- Fix some errors
  - setting.json was wrong
  - lambda didn't have permission to put object to s3
  - url was wrongly appended
- Enable client, server build cache for ci
- namui-prebuilt Typography more functions

Test url: https://jljzah2litkb6w3seks327wdti0hsooj.lambda-url.ap-northeast-2.on.aws/

### CODE REVIEW

File: luda-editor/client/src/app/login.rs [hunk 1]
```diff
@@ -62,14 +62,12 @@ pub fn check_session_id() {
...
+    crate::RPC.set_session_id(session_id);
     match crate::RPC
         .validate_session(rpc::validate_session::Request {})
         .await
     {
-        Ok(_) => {
-            crate::RPC.set_session_id(session_id);
-            Ok(true)
-        }
+        Ok(_) => Ok(true),
```
Comment (namse) [inline luda-editor/client/src/app/login.rs:70]: I wrongly moved it in previous PR.

File: luda-editor/server/src/services/image/mod.rs [hunk 2]
```diff
@@ -59,104 +96,21 @@ impl rpc::ImageService<SessionDocument> for ImageService {
...
-    .map(|c| Err(LabelError::InvalidCharacter(c.to_string())))
-    .unwrap_or(Ok(()))
-}
-
-#[derive(Debug)]
-enum LabelError {
-    InvalidCharacter(String),
-    TooLong,
-}
-crate::simple_error_impl!(LabelError);
```
Comment (namse) [inline luda-editor/server/src/services/image/mod.rs:162]: I decide not to use s3 directly for image. I save image metadata(id, labels) to dynamodb and store image binary data to s3.

File: luda-editor/server/src/utils/mod.rs [hunk 3]
```diff
@@ -9,3 +9,43 @@ macro_rules! simple_error_impl {
...
+    #[test]
+    fn test_append_slash() {
+        assert_eq!(append_slash!["a", "b"], "a/b");
+        assert_eq!(append_slash!["a/", "b"], "a/b");
+        assert_eq!(append_slash!["a", "/b"], "a/b");
+        assert_eq!(append_slash!["a/", "/b"], "a/b");
+        assert_eq!(append_slash!["a/", "/b/"], "a/b/");
+        assert_eq!(append_slash!["a/", "b/"], "a/b/");
+    }
+}
```
Comment (bigfoodK) [inline luda-editor/server/src/utils/mod.rs:51]: It's duplicated with `luda-editor/client/src/storage/resource_url.rs`.
Do you think it is overhead to split macro into different crate?

File: luda-editor/client/src/pages/sequence_edit_page/loaded/mod.rs [hunk 4]
```diff
@@ -50,20 +67,39 @@ impl LoadedSequenceEditorPage {
             });
             line_text_inputs
         };
+        start_load_recent_selected_image_ids();
         Self {
-            project_id,
+            project_id: project_id.clone(),
```
[the remark this answers is not in the document]
Comment (namse) [reply luda-editor/client/src/pages/sequence_edit_page/loaded/mod.rs]: Fixed all explicit id clone

File: luda-editor/server/src/utils/mod.rs (hunk 3 above)
Comment (namse) [reply luda-editor/server/src/utils/mod.rs:51]: I'm not sure right now. It's the first and single case so I will make a decision when I make next second duplicated code.

**Review by bigfoodK — APPROVED** [review]

**Pull request merged**